README.md

```diff
@@ -182,15 +182,15 @@ Note: For session cookies to work correctly, add the below entries to your hosts
 
 ### Development Tasks
 
-- `yarn run test` or `npm run test` to run karma tests
-- `yarn run lint` or `npm run lint` to run eslint
-- `yarn run build` or `npm run build` to generate minified output files. These files are output to `/dist`. `common.js` must be included before any of the other JS files.
-- `yarn run build:analyze` or `npm run build:analyze` to open a visualization of bundle sizes after building
+- `yarn test` to run karma tests
```

Comment:
Are these still Karma, or are they Jest?

Reply:
Good point, it's Jest now. ESLint is still kind of configured, but `yarn lint` runs standard, so I should update that too.